pkgs/by-name/cl/cling/package.nix

```diff
@@ -34,41 +34,41 @@
 let
   stdenv = clangStdenv;
 
-  # The patched clang lives in the LLVM megarepo
-  clangSrc = fetchFromGitHub {
+  version = "1.2";
+
+  clingSrc = fetchFromGitHub {
     owner = "root-project";
-    repo = "llvm-project";
-    # cling-llvm13 branch
-    rev = "3610201fbe0352a63efb5cb45f4ea4987702c735";
-    sha256 = "sha256-Cb7BvV7yobG+mkaYe7zD2KcnPvm8/vmVATNWssklXyk=";
-    sparseCheckout = [ "clang" ];
+    repo = "cling";
+    rev = "v${version}";
+    sha256 = "sha256-ay9FXANJmB/+AdnCR4WOKHuPm6P88wLqoOgiKJwJ8JM=";
   };
 
-  llvm = llvmPackages_13.llvm.override { enableSharedLibraries = false; };
+  llvm = llvmPackages_18.llvm.override { enableSharedLibraries = false; };
 
-  unwrapped = stdenv.mkDerivation rec {
+  unwrapped = stdenv.mkDerivation {
     pname = "cling-unwrapped";
-    version = "1.0";
-
-    src = "${clangSrc}/clang";
+    inherit version;
 
-    clingSrc = fetchFromGitHub {
+    src = fetchFromGitHub {
       owner = "root-project";
-      repo = "cling";
-      rev = "v${version}";
-      sha256 = "sha256-Ye8EINzt+dyNvUIRydACXzb/xEPLm0YSkz08Xxw3xp4=";
+      repo = "llvm-project";
+      # cling-llvm18 branch
+      rev = "156e947058a46ecc1785f98aa9abb8cbfaa45aa7";
```

Comment:
Maybe a good idea to use one of their tags, as in https://github.com/NixOS/nixpkgs/pull/439740? (Not a blocker, as not a new issue.)